README.md

```diff
@@ -1,6 +1,9 @@
-# Payone Module
+# PayoneModule
```

Comment:
missing space!

Reply:
@dereuromark Do we have an article with described requirements related to a module description text?

Reply:
See existing module generator in Development.

ModuleName Module

as for all other 5xx modules.

Reply:
Thanks.

Reply:
Fixed.